ethexe/network/src/lib.rs

```diff
@@ -67,11 +67,11 @@ const MAX_ESTABLISHED_INCOMING_PER_PEER_CONNECTIONS: u32 = 1;
 const MAX_ESTABLISHED_OUTBOUND_PER_PEER_CONNECTIONS: u32 = 1;
 const MAX_ESTABLISHED_INCOMING_CONNECTIONS: u32 = 100;
 
-#[derive(Clone, Eq, PartialEq)]
+#[derive(Eq, PartialEq)]
 pub enum NetworkEvent {
     DbResponse {
         request_id: db_sync::RequestId,
-        result: Result<db_sync::Response, (db_sync::OngoingRequest, db_sync::RequestFailure)>,
+        result: Result<db_sync::Response, (db_sync::RetriableRequest, db_sync::RequestFailure)>,
```

Comment:
we may want to insert recent failure directly into retriable request

Reply:
`RequestFailure` is in event, so I keep it out of `RetriableRequest` for consistency